src/Dapr.Client/SaveBulkStateItem.cs

```diff
@@ -0,0 +1,65 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2021 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Dapr.Client
+{
+    /// <summary>
+    /// Represents a state object used for bulk delete state operation
+    /// </summary>
+    public readonly struct SaveBulkStateItem
```

Comment:
Nit, but I think this should be named `StateItem` or `SaveStateRequest`. This struct isn't really bulk in anyway, it's just used in a List to call the bulk API.

Reply:
Yeah, I tried with that but then I got a conflict for `StateItem ` : https://github.com/dapr/dotnet-sdk/blob/01cdc89710c16712442474afea97693f22c5852b/src/Dapr.Client/DaprClientGrpc.cs#L619 

Same with `SaveStateRequest`: https://github.com/dapr/dotnet-sdk/blob/01cdc89710c16712442474afea97693f22c5852b/src/Dapr.Client/DaprClientGrpc.cs#L758